src/app/components/Shared/progressbar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export default function Progressbar({
```

Comment:
For the current styling I think we can just use the "native" progress element? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

Reply:
Oh I didn't know there is one. cool!